client/src/App.jsx

```diff
@@ -37,8 +37,8 @@ class App extends React.Component {
 		$.get(`http://localhost:3003/${id}/images`, (data) => {
 			console.log(data);
 			this.setState({
-				mainImage: data[id][0],
-				imagesList: data[id],
+				mainImage: data['images'][0],
+				imagesList: data['images'],
```

Comment:
are you hardcoding this id here because there is no image service running?

Reply:
probably it doesn't matter as this is not part of the db script - lol

Reply:
Oh yeah this is the front end code. I made some modifications to the response object, so all entries have the same key's. Previously, she had the response with different key value's . I'm hardcoding the word 'images' to get the correct response urls.